bigbluebutton-html5/imports/ui/components/nav-bar/nav-bar-graphql/talking-indicator/component.tsx

```diff
@@ -62,66 +61,63 @@ const TalkingIndicator: React.FC<TalkingIndicatorProps> = ({
   isModerator,
 }) => {
   const intl = useIntl();
-  const talkingElements = useMemo(() => {
-    return talkingUsers.map((user) => {
-      const {
-        talking,
-        muted,
-        userId,
-        user: { color, speechLocale },
-        // transcribing,
-      } = user;
-
-      const name = user.user?.name;
-      const ariaLabel = intl.formatMessage(talking
-        ? intlMessages.isTalking : intlMessages.wasTalking, {
-        0: name,
-      });
-      let icon = talking ? 'unmute' : 'blank';
-      icon = muted ? 'mute' : icon;
-      return (
-        <Styled.TalkingIndicatorWrapper
+  const talkingElements = useMemo(() => talkingUsers.map((talkingUser: Partial<UserVoice>) => {
+    const {
+      talking,
+      muted,
+      user: { color, speechLocale, userId } = {} as User,
```

Comment:
You can use [Partial](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) in that case

Reply:
Altered to partial